packages/components/Atoms/Link/Link.vue

```diff
@@ -69,7 +69,7 @@ export default {
     }
   },
   created: function () {
-    if (!isAnchorLink(this.href) && isRelativeUrl(this.href)) {
+    if (!isAnchorLink(this.href) && !this.linkTarget && isRelativeUrl(this.href)) {
```

Comment:
@dylankelly @waitingallday this is the new line that fixes internal links not opening in new tabs if target is set